pages/sdk/[...slug].vue

```diff
@@ -52,11 +52,10 @@ const links = computed(() =>
   ].filter(Boolean)
 );
 </script>
-
 <template>
   <UPage v-if="page">
     <UPageHeader
-      :title="page.title"
+      :title="page.menuTitle || page.title"
```

Comment:
I like the idea, but it feels a little too confusing to have multiple title labels. For accessibility and consistency, we should have the link in the navigation match the title on the page. Let's leave the title to be 'Introduction' and drop the menuTitle for now.

Reply:
Done